spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/config/EurekaConfigServerInstanceProvider.java

```diff
@@ -31,6 +31,9 @@
 import org.springframework.cloud.netflix.eureka.EurekaServiceInstance;
 import org.springframework.http.HttpStatus;
 
+/**
+ * @author puppy
```

Comment:
Please use your full name and last name.

Reply:
Modified